infrastructure/scripts/update-github-actions-permissions.sh

```
@@ -0,0 +1,359 @@
+        return 1
+    fi
+}
+
+# Function to get current role policies
+get_current_policies() {
+    local role_name="$1"
+    
+    print_info "Getting current attached policies for role: $role_name"
+    
+    # Get managed policies
+    local managed_policies=$(aws iam list-attached-role-policies \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)